### ISSUE TITLE
SafeMath not needed since Solidity 0.8.0?

### ISSUE THREAD

**Issue by aspiers** [sigp/solidity-security-blog#13]
https://docs.soliditylang.org/en/latest/080-breaking-changes.html says:

> Arithmetic operations revert on underflow and overflow. You can use `unchecked { ... }` to use the previous wrapping behaviour.

However OpenZeppelin are keeping it for reasons explained here:

https://github.com/OpenZeppelin/openzeppelin-contracts/issues/2465

It would be helpful to update this blog to reflect that.

**Comment by AgeManning** [comment]
Yep, thanks for pointing this out. Its on our to-do list to update the entire list based on latest solidity.
